va/va.go

```diff
@@ -1038,17 +1038,27 @@ func (va *ValidationAuthorityImpl) getCAASet(ctx context.Context, hostname strin
 	return parseResults(results)
 }
 
-func (va *ValidationAuthorityImpl) checkCAARecords(ctx context.Context, identifier core.AcmeIdentifier) (present, valid bool, err error) {
+func (va *ValidationAuthorityImpl) checkCAARecords(ctx context.Context, identifier core.AcmeIdentifier, challengeType string) (present, valid bool, err error) {
 	hostname := strings.ToLower(identifier.Value)
 	caaSet, err := va.getCAASet(ctx, hostname)
 	if err != nil {
 		return false, false, err
 	}
-	present, valid = va.validateCAASet(caaSet)
+	present, valid = va.validateCAASet(caaSet, challengeType)
 	return present, valid, nil
 }
 
-func (va *ValidationAuthorityImpl) validateCAASet(caaSet *CAASet) (present, valid bool) {
+func containsMethod(commaSeparatedMethods string, method string) bool {
```

Comment:
small nit: this could be `func containsMethod(commaSeparatedMethods, method string)` instead of duplicating the `string` portion.